package.json

```diff
@@ -30,23 +30,22 @@
     "lint": "eslint --ext .js,.vue --ignore-path .gitignore .",
     "lint:fix": "npm run lint -- --fix",
     "size": "npm run build && size-limit",
-    "stack:down": "docker-compose -f ./docker/stack/docker-compose.yml down",
-    "stack:up:detach": "docker-compose -f ./docker/stack/docker-compose.yml build && docker-compose -f ./docker/stack/docker-compose.yml up -d",
-    "stack:up": "docker-compose -f ./docker/stack/docker-compose.yml build && docker-compose -f ./docker/stack/docker-compose.yml up",
```

Comment:
Why has the stack up command been removed? it's been replaced by test:stack:start ?

Reply:
Yes.